src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -107,12 +107,12 @@
     ]},
 
     {node_metrics, [
-        {2, file_descriptors_open_limit, gauge, "Open file descriptors limit", fd_total},
-        {2, tcp_sockets_open_limit, gauge, "Open TCP sockets limit", sockets_total},
+        {2, erlang_process_max_fds, gauge, "Open file descriptors limit", fd_total},
```

Comment:
If this is the same as what you'd get elsewhere go with process_max_fds, users often will have alerts on the standard exports to know when they're short on fds: https://prometheus.io/docs/instrumenting/writing_clientlibs/#process-metrics